xc/device.ml

```diff
@@ -2017,14 +2017,13 @@ module Dm_Common = struct
                |> String.concat ",")
           in
           match x.implementation with
-          | Nvidia {virtual_pci_address; type_id = Some type_id; uuid = Some uuid; extra_args} ->
-            make [type_id; Xcp_pci.string_of_address virtual_pci_address; uuid; extra_args]
+          (* 1. Upgrade case, migrate from a old host with old vGPU having config_path
+           * 2. Legency case, run with old Nvidia host driver *)
           | Nvidia {virtual_pci_address; config_file = Some config_file; extra_args} ->
```

Comment:
I think both cases should have a `debug` log line:
* `debug "NVidia vGPU config: using config file %s" confile_file`
* `debug "NVidia vGPU config: using type id %s" type_id`
If possible, including the UUID of the vGPU would be good.

Reply:
updated.